tests/test_benchmark.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (C) 2019 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+"""
+pytest-benchmark benchmarks for conda-content-trust.
+"""
+import datetime
+
+import cryptography.exceptions
+import pytest
+
+from conda_content_trust.authentication import verify_signature
+from conda_content_trust.common import (  # Putting this entire import list here as a sort of "to do" list
+    PublicKey,
+    checkformat_any_signature,
+    checkformat_byteslike,
+    checkformat_delegating_metadata,
+    checkformat_delegation,
+    checkformat_delegations,
+    checkformat_expiration_distance,
+    checkformat_gpg_signature,
+    checkformat_hex_key,
+    checkformat_hex_string,
+    checkformat_list_of_hex_keys,
+    checkformat_signature,
+    checkformat_string,
+)
+
+from .test_authentication import (
+    REG__MESSAGE_THAT_WAS_SIGNED,
+    REG__PUBLIC_BYTES,
+    REG__SIGNATURE_HEX,
+    TEST_ROOT_MD_V1,
+)
+from .test_common import SAMPLE_GPG_SIG
+
+SAMPLE_HEX_KEY_1 = "abcde123" * 8
+SAMPLE_HEX_KEY_2 = "deadbeef" * 8
+
+
+def bench_checkformat_hex_string():
+    hex_string = "0123456789abcdef" * 4
+    checkformat_hex_string(hex_string)
+
+
+def test_benchmark_checkformat_hex_string(benchmark):
+    benchmark(bench_checkformat_hex_string)
```

Comment:
`benchmark` works like Python's `functools.partial` and accepts args and kwargs:

```suggestion
def test_benchmark_checkformat_hex_string(benchmark):
    hex_string = "0123456789abcdef" * 4
    benchmark(checkformat_hex_string, hex_string)
```